model/user.go

```diff
@@ -457,31 +457,25 @@ func IsInRole(userRoles string, inRole string) bool {
 		if r == inRole {
 			return true
 		}
-
 	}
 
 	return false
 }
 
 func (u *User) IsSSOUser() bool {
-	if u.AuthService != "" && u.AuthService != USER_AUTH_SERVICE_EMAIL {
-		return true
-	}
-	return false
+	return u.AuthService != "" && u.AuthService != USER_AUTH_SERVICE_EMAIL
 }
 
 func (u *User) IsOAuthUser() bool {
-	if u.AuthService == USER_AUTH_SERVICE_GITLAB {
-		return true
-	}
-	return false
+	return u.AuthService == USER_AUTH_SERVICE_GITLAB
```

Comment:
Yeah, you are right actually. However, in many places in the codebase we treat "Google Auth" and "O365 Auth" as being their own independent authentication schemes. However, here I'm not sure what the expected return value of this function would be based on it's name: just Gitlab, or include Google/O365 as well. Prior to this patch `IsOAuthUser()` is only called in 1 place in the codebase anyway. I'll leave it until someone more familiar with the authentication parts of the codebase is around to answer this decisively.